requirements.txt

```
@@ -9,7 +9,7 @@ fauxfactory==3.1.2
 jinja2==3.1.6
 manifester==0.2.10
 navmazing==1.3.0
-productmd==1.43
+productmd==1.44
```

Fixes https://github.com/SatelliteQE/robottelo/issues/18239
```suggestion
productmd==1.45
```